docs/running-in-ci.md

```diff
@@ -0,0 +1,239 @@
+# Running in CI
+
+[TOC]
+
+The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a free [Semgrep Community](https://semgrep.dev/manage) or paid Semgrep Team account. `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
+
+!!! danger
+    `SEMGREP_TOKEN` is a secret value: DO NOT HARDCODE IT AND LEAK CREDENTIALS. Use your CI provider's secret or environment variable management feature to store it.
+
+Semgrep can seamlessly integrate into your CI pipeline using GitHub Actions or GitLab CI.
+
+## GitHub Actions
+
+The easiest way to set up the Semgrep CI integration in GitHub
+is via the [Semgrep Community](https://semgrep.dev/manage/) app.
```

Comment:
I think we'll want to move towards Semgrep App as the descriptor, reserving Community and Team for discussion of specific tiers. 

I've created a tracking ticket to make that rename consistent across the docs. CC @pabloest